pom.xml

```diff
@@ -29,7 +29,7 @@
 
     <groupId>com.github.cafdataprocessing</groupId>
     <artifactId>worker-workflow-aggregator</artifactId>
-    <version>4.4.0-SNAPSHOT</version>
+    <version>5.0.0-SNAPSHOT</version>
```

Comment:
Breaking change declared but not documented:
[release-notes-5.0.0.md](https://github.com/CAFDataProcessing/worker-workflow/blob/44fe4f6d23b819b894e7d1ca7660a14284989592/release-notes-5.0.0.md)

Reply:
Sorry, I had updated them but forgot to push the changes last night.